pkg/cybr/api/auth.go

```diff
@@ -23,9 +30,23 @@ func (c *Client) Logon(req LogonRequest) error {
 		return err
 	}
 
+	// Handle cyberark, ldap, and radius push & append methods authentication methods
 	url := fmt.Sprintf("%s/PasswordVault/api/auth/%s/logon", c.BaseURL, c.AuthType)
 	token, err := httpJson.SendRequestRaw(url, "POST", "", req, c.InsecureTLS, c.Logger)
 	if err != nil {
+		if strings.Contains(err.Error(), "ITATS542I") {
+			// Get secret value from STDIN
+			fmt.Print("Enter one-time passcode: ")
+			byteOTPCode, err := terminal.ReadPassword(int(syscall.Stdin))
+			req.Password = string(byteOTPCode)
+			fmt.Println()
+			if err != nil {
+				log.Fatalln("An error occurred trying to read one-time passcode from " +
+					"Stdin. Exiting...")
+			}
+			token, err = httpJson.SendRequestRaw(url, "POST", "", req, c.InsecureTLS, c.Logger)
+			return nil
+		}
```

Comment:
Line of sight is a little wonky here. I think we need some refactoring. Also prompting for interactive logon in the api package makes me a little hesitant. Let me chew on this one for a little.